src/pages/tabs/tabs.ts

```diff
@@ -12,6 +14,7 @@ export class TabsPage {
   tab1Root = HomePage;
   tab2Root = AboutPage;
   tab3Root = ContactPage;
+  tab4Root = MarketPage;
```

Comment:
The class property 'tab4Root' must be marked either 'private', 'public', or 'protected'<br>expected member-variable-declaration: 'tab4Root' to have a typedef